crates/kornia-tensor-ops/src/ops.rs

```diff
@@ -131,6 +131,198 @@ where
         .map_err(|_| TensorOpsError::ShapeMismatch(tensor.shape.to_vec(), other.shape.to_vec()))
 }
 
+/// Apply the power function to the pixel data.
+///
+/// # Arguments
+///
+/// * `n` - The power to raise the pixel data to.
+///
+/// # Returns
+///
+/// A new image with the pixel data raised to the power.
+pub fn powi<T, const N: usize, A>(tensor: &Tensor<T, N, A>, n: i32) -> Tensor<T, N, A>
+where
+    T: Float + Clone,
+    A: TensorAllocator + Clone + 'static,
+{
+    tensor.map(|x| x.powi(n))
+}
+
+/// Compute absolute value of the pixel data.
+///
+/// # Returns
+///
+/// A new image with the pixel data absolute value.
+pub fn abs<T, const N: usize, A>(tensor: &Tensor<T, N, A>) -> Tensor<T, N, A>
+where
+    T: Float + Clone,
+    A: TensorAllocator + Clone + 'static,
+{
+    tensor.map(|x| x.abs())
+}
+
+/// Compute the mean of the pixel data.
+///
+/// # Returns
+///
+/// The mean of the pixel data.
+pub fn mean<T, const N: usize, A>(tensor: &Tensor<T, N, A>) -> Result<T, TensorError>
+where
+    T: Float + Clone,
+    A: TensorAllocator + Clone + 'static,
+{
+    let data_acc = tensor.as_slice().iter().fold(T::zero(), |acc, &x| acc + x);
+    let mean = data_acc / T::from(tensor.as_slice().len()).ok_or(TensorError::CastError)?;
+
+    Ok(mean)
+}
+/// Perform an element-wise addition on two tensors.
+///
+/// # Arguments
+///
+/// * `other` - The other tensor to add.
+///
+/// # Returns
+///
+/// A new `Tensor` instance.
+///
+/// # Example
+///
+/// ```
+/// use kornia_tensor::{Tensor, CpuAllocator};
+/// use kornia_tensor_ops::ops::add;
+///
+/// let data1: Vec<u8> = vec![1, 2, 3, 4];
+/// let t1 = Tensor::<u8, 1, CpuAllocator>::from_shape_vec([4], data1, CpuAllocator).unwrap();
+///
+/// let data2: Vec<u8> = vec![1, 2, 3, 4];
+/// let t2 = Tensor::<u8, 1, CpuAllocator>::from_shape_vec([4], data2, CpuAllocator).unwrap();
+///
+/// let t3 = add(&t1,&t2).unwrap();
+/// assert_eq!(t3.as_slice(), vec![2, 4, 6, 8]);
+/// ```
+pub fn add<T, const N: usize>(
```

Comment:
```suggestion
fn add<T, const N: usize>(
```
wondering if this functions should be just an internal implementation for the high level api `t1.add(t2)?` ?